instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -50,6 +50,17 @@ def hello():
     application = get_wsgi_application()
     application = OpenTelemetryMiddleware(application)
 
+
+Usage in Production Server (gunicorn)
+------------------------------------
+
+Modify the ``gunicorn.config.py`` file as shown below.
+
+.. code-block:: python
+
+    def post_fork(server, worker):
+        worker(DjangoInstrumentor().instrument())
```

Comment:
1. As far as I remember, tracing as a whole needs to be setup in post_fork including creating the tracer provider, span processors, instrumentation etc. Especially if batch span processor is used. Any reason to only mention instrumentation here?

Reply:
Are you suggesting that I create a pipeline in post_fork? I was thinking more of using opentelemetry as an agent. I could be wrong.

Reply:
I see that you recommended a pipeline. Let me put that in effect.